dependencies.json

```diff
@@ -0,0 +1,11 @@
+{
+    "recipesEndpoint": "",
+    "packages": [
+        {
+            "requirement": "dev-ibx-8811-SA-sessions-sharing as 5.0.x-dev",
+            "repositoryUrl": "https://github.com/ibexa/core",
+            "package": "ibexa/core",
+            "shouldBeAddedAsVCS": true
```

Comment:
AFAIR, value for this parameter should be set to false because `ibexa/core` has public access. This should fix red CI.

Reply:
https://github.com/ibexa/http-cache/compare/d4fd5539798ef12f5a98e5ddbd91de0b243493bd..2d2012ba1747a0d8ede0175dc25f68cf22d306cf - let us wait for the CI then, thanks.